pkgs/top-level/all-packages.nix

```diff
@@ -5626,6 +5626,8 @@ with pkgs;
 
   gti = callPackage ../tools/misc/gti { };
 
+  harlequin = callPackage ../by-name/ha/harlequin { };
+
```

Comment:
rename `pkgs/by-name/ha/harlequin/default.nix` -> `pkgs/by-name/ha/harlequin/package.nix` and these two lines can be deleted.

Reply:
Nice, done now